requirements.txt

```diff
@@ -1 +1 @@
-aiohttp>=3.6.0,<3.8.0
+aiohttp>=3.6.0,<4.0.0
```

Comment:
Would be better to bump every new update to make sure users aren't using non working versions
```suggestion
aiohttp>=3.6.0,<3.8.1
```

Reply:
semver means that there won't be breaking changes before the next major

Reply:
changed it to 3.9